package/helm-cmd

```diff
@@ -15,6 +15,10 @@ for i in $(seq 1 20); do
 done
 
 for i in operation*; do
+    kustomization=$(echo $i.yaml | sed "s/operation/kustomization/g")
```

Comment:
What's the purpose of this?

Reply:
Identifying what the kustomization file will be based off the operation file. So if it is operation001.yaml then if kustomize is being used, the kustomization file would have been passed as kustomization001.yaml.

Reply:
I'm confused about this, is this `operationxyz` thing a specific use-case we are hardcoding somewhere ? We should add a comment here, as its quite specific logic.

Reply:
It's done here: https://github.com/rancher/rancher/pull/35202/files#diff-16ed52522341783cf957d7732c82a4d17845f6242a6f90c9bbd5f5cc90c0890bR476-R479. I'm not sure what you mean by hardcoding; this is being passed in by a secret just like operation.

Reply:
@cbron Added comments, also added some below.

Reply:
Thats what I was looking for